### ISSUE TITLE
Default colors hard to read on some terminals

### ISSUE THREAD

**Issue by erezsh** [ewels/rich-click#72]
For example, this is a screenshot on Windows, using "Windows Terminal".

![image](https://user-images.githubusercontent.com/731464/170215677-16abb9b5-b855-4be9-9a63-2b4680f621ea.png)

And this one is taken from a regular cmd:

![image](https://user-images.githubusercontent.com/731464/170216047-f313bdf1-f771-41e8-b058-b6bf0c9ad340.png)

**Comment by ewels** [comment]
Yeah, I saw in a comment recently that the colours aren't ideal in light colour themes either. Picking colours that work everywhere is difficult! I want to use colours as much as possible to make content visually distinct (that's kind of the whole point of the package).

Any suggestions for improvements?

**Comment by ewels** [comment]
NB: Remember that you can overwrite the default styles / colours for any element with rich-click. See the end of the readme. But good defaults is obviously the best solution here.

**Comment by erezsh** [comment]
I think playing it safe is the best approach for default themes. It's better to risk something being less distinct, than it being hard to read even when your attention is on it.

For dark, I would go with lighter colors (maybe pastel-oriented). I don't know about the light theme, but I guess it needs stronger darker colors.

p.s. note that the first one supports true-color (or close enough), so you can play with shades. The second one is 16 colors, so nothing will make it pretty, but it needs to be cyan or light gray, not blue.

Anyway, just me 2 cents. I know it's hard to get it right for every terminal.

**Comment by ewels** [comment]
Yup, but what would be playing it safe other than uncoloured text?

Light terminal themes use the same rich-click theme as everything else, so the colours have to work on both light and dark terminal themes. So using lighter paster-colours would be worse for those users.

**Comment by erezsh** [comment]
Oh, I see the problem now. I assumed there was some way to detect it.

This sounds like a fundamental problem. Maybe `rich` should implement something like this https://github.com/dalance/termbg

**Comment by ewels** [comment]
Hmm 🤔 Maybe it does already - I haven't looked for it. Wouldn't be the first time 😅

**Comment by erezsh** [comment]
Seems that it doesn't right now. But it never hurts to ask - https://github.com/Textualize/rich/discussions/1170

**Comment by dwreeves** [comment]
Marking this as won't fix.

The reasoning is: The default 16 ANSI colors should be configured on a terminal-by-terminal basis, and there are modern terminals available in Windows that allow for this level of customization. We also do not want to disrupt defaults for users who have come to expect them.

That said, 1.9 should be introducing themes, and themes will be able to be set either via the `rich_config()` or via environment variables, and this can assist in providing control of colors to both developers and end-users.
